code/controllers/subsystem/monitor.dm

```diff
@@ -42,7 +42,7 @@ SUBSYSTEM_DEF(monitor)
 	var/list/b17_in_use = list()
 
 /datum/controller/subsystem/monitor/Initialize()
-	RegisterSignals(SSdcs, list(COMSIG_GLOB_OPEN_TIMED_SHUTTERS_LATE, COMSIG_GLOB_OPEN_SHUTTERS_EARLY, COMSIG_GLOB_CRASH_SHIP_LANDED), PROC_REF(set_groundside_calculation))
+	RegisterSignals(SSdcs, list(COMSIG_GLOB_OPEN_TIMED_SHUTTERS_LATE, COMSIG_GLOB_OPEN_SHUTTERS_EARLY, COMSIG_GLOB_OPEN_TIMED_SHUTTERS_XENO_HIVEMIND, COMSIG_GLOB_TADPOLE_LANDED_OUT_LZ, COMSIG_GLOB_TADPOLE_RAPPEL_DEPLOYED_OUT_LZ, COMSIG_GLOB_DROPPOD_LANDED, COMSIG_GLOB_CRASH_SHIP_LANDED), PROC_REF(set_groundside_calculation))
```

Comment:
COMSIG_GLOB_OPEN_TIMED_SHUTTERS_XENO_HIVEMIND seems to be entirely unused, and needs to be just deleted.
COMSIG_GLOB_DROPPOD_LANDED is sent even if the pod is landing inside the LZ, which presumably you don't want to trigger this.
Several of these are also sent multiple times - set_groundside_calculation needs to unregister these signals.